NEWS.md

```diff
@@ -87,6 +87,15 @@
   this doesn't work properly as it might result in leftover files in the install directory from a previous
   installation, so we have to empty the directory before reusing it, invalidating the caching, so the option
   was removed.
+- Build scripts are now executed on the host. See the `SCRIPTS` section
```

Comment:
This description should be beefed up, saying that existing scripts need to be evaluated to ensure what they do is safe, and saying that as a safeguard the filesystem is mostly r/o. Yes it's in the manpage too, but given the potential for damage, it's better to be repetitive I think